jQuery/styles.css

```diff
@@ -0,0 +1,30 @@
+body {
```

Comment:
Unless you're using a reset, targeting elements generically is generally stupid and bad.